enzy_htp/structure/chain.py

```diff
@@ -1,7 +1,6 @@
 """Definition for the Chain class. Chains primarily store Residue() objects and organize them
 within the overall structure of an enzyme.
-Author: Qianzhen (QZ) Shao <shaoqz@icloud.com>
-Author: Chris Jurich <chris.jurich@vanderbilt.edu>
+Author: Qianzhen (QZ) Shao <shaoqz@icloud.com> Author: Chris Jurich <chris.jurich@vanderbilt.edu>
```

Comment:
Why is this formatting lol?

Reply:
Quite odd. Probably caused by a defective vim shortcut or something.